storage/reads/group_resultset.go

```diff
@@ -121,7 +121,10 @@ func (g *groupResultSet) Next() GroupCursor {
 // the time range of the query.
 func (g *groupResultSet) seriesHasPoints(row *SeriesRow) bool {
 	// TODO(sgc): this is expensive. Storage engine must provide efficient time range queries of series keys.
-	cur := g.arrayCursors.createCursor(*row)
+	cur, err := g.arrayCursors.createCursor(*row)
```

Comment:
So this is a lost error; unreported, never makes it out of this method scope.  I'm not sure how to handle this, but because there is also a panic on line 148, you may need to return a `(bool, error)` from this method to get this error and the (soon-to-be-eliminated) panic out to the user.